base/task.jl

```diff
@@ -55,6 +55,20 @@ end
 
 Wrap an expression in a [`Task`](:class:`Task`) without executing it, and return the [`Task`](:class:`Task`). This only
 creates a task, and does not run it.
+
+```jldoctest
+julia> a1() = det(rand(1000, 1000));
+
+julia> b = @task a1();
+
+julia> istaskstarted(b)
+false
+
+julia> schedule(b);
+
+julia> istaskdone(b)
```

Comment:
A `yield()` should do it. `schedule` adds it to the end of runnable task queue. Yielding will enable it to be run and hence complete.